# This file is part of HiGFXback

# requires
REQUIRES="autotools-wrappers-lt-build libffi-build pcre-build python3-build util-linux-build"

pkg-config --exists --print-errors $REQUIRES || exit 1

# configure (ac-2.69; am-1.15; lt-2.4.6)
libtoolize -c -f; aclocal; autoheader; autoconf; automake -a -c
./configure --with-python=/bin/python3 --prefix=

# build
make

# install
install -d $DESTDIR/bin
install gio/.libs/gapplication $DESTDIR/bin
install gio/.libs/gdbus $DESTDIR/bin
install gio/.libs/gio $DESTDIR/bin
install gio/.libs/gio-querymodules $DESTDIR/bin
install gio/.libs/glib-compile-resources $DESTDIR/bin
install gio/.libs/glib-compile-schemas $DESTDIR/bin
install gio/.libs/gresource $DESTDIR/bin
install gio/.libs/gsettings $DESTDIR/bin
install gio/gdbus-2.0/codegen/gdbus-codegen $DESTDIR/bin
install gobject/glib-mkenums $DESTDIR/bin
install gobject/.libs/glib-genmarshal $DESTDIR/bin
install -d $DESTDIR/include/gio-unix-2.0/gio
install -m 644 gio/gdesktopappinfo.h $DESTDIR/include/gio-unix-2.0/gio
install -m 644 gio/gunixconnection.h $DESTDIR/include/gio-unix-2.0/gio
install -m 644 gio/gunixfdlist.h $DESTDIR/include/gio-unix-2.0/gio
install -m 644 gio/gunixfdmessage.h $DESTDIR/include/gio-unix-2.0/gio
install -m 644 gio/gunixinputstream.h $DESTDIR/include/gio-unix-2.0/gio
install -m 644 gio/gunixmounts.h $DESTDIR/include/gio-unix-2.0/gio
install -m 644 gio/gunixoutputstream.h $DESTDIR/include/gio-unix-2.0/gio
install -m 644 gio/gunixsocketaddress.h $DESTDIR/include/gio-unix-2.0/gio
install -d $DESTDIR/include/glib-2.0
install -m 644 glib/glib.h $DESTDIR/include/glib-2.0
install -m 644 glib/glibconfig.h $DESTDIR/include/glib-2.0
install -m 644 glib/glib-object.h $DESTDIR/include/glib-2.0
install -m 644 glib/glib-unix.h $DESTDIR/include/glib-2.0
install -m 644 gmodule/gmodule.h $DESTDIR/include/glib-2.0
install -d $DESTDIR/include/glib-2.0/gio
install -m 644 gio/gaction.h $DESTDIR/include/glib-2.0/gio
install -m 644 gio/gactiongroup.h $DESTDIR/include/glib-2.0/gio
install -m 644 gio/gactiongroupexporter.h $DESTDIR/include/glib-2.0/gio
install -m 644 gio/gactionmap.h $DESTDIR/include/glib-2.0/gio
install -m 644 gio/gappinfo.h $DESTDIR/include/glib-2.0/gio
install -m 644 gio/gapplication.h $DESTDIR/include/glib-2.0/gio
install -m 644 gio/gapplicationcommandline.h $DESTDIR/include/glib-2.0/gio
install -m 644 gio/gasyncinitable.h $DESTDIR/include/glib-2.0/gio
install -m 644 gio/gasyncresult.h $DESTDIR/include/glib-2.0/gio
install -m 644 gio/gbufferedinputstream.h $DESTDIR/include/glib-2.0/gio
install -m 644 gio/gbufferedoutputstream.h $DESTDIR/include/glib-2.0/gio
install -m 644 gio/gbytesicon.h $DESTDIR/include/glib-2.0/gio
install -m 644 gio/gcancellable.h $DESTDIR/include/glib-2.0/gio
install -m 644 gio/gcharsetconverter.h $DESTDIR/include/glib-2.0/gio
install -m 644 gio/gcontenttype.h $DESTDIR/include/glib-2.0/gio
install -m 644 gio/gconverter.h $DESTDIR/include/glib-2.0/gio
install -m 644 gio/gconverterinputstream.h $DESTDIR/include/glib-2.0/gio
install -m 644 gio/gconverteroutputstream.h $DESTDIR/include/glib-2.0/gio
install -m 644 gio/gcredentials.h $DESTDIR/include/glib-2.0/gio
install -m 644 gio/gdatagrambased.h $DESTDIR/include/glib-2.0/gio
install -m 644 gio/gdatainputstream.h $DESTDIR/include/glib-2.0/gio
install -m 644 gio/gdataoutputstream.h $DESTDIR/include/glib-2.0/gio
install -m 644 gio/gdbusactiongroup.h $DESTDIR/include/glib-2.0/gio
install -m 644 gio/gdbusaddress.h $DESTDIR/include/glib-2.0/gio
install -m 644 gio/gdbusauthobserver.h $DESTDIR/include/glib-2.0/gio
install -m 644 gio/gdbusconnection.h $DESTDIR/include/glib-2.0/gio
install -m 644 gio/gdbuserror.h $DESTDIR/include/glib-2.0/gio
install -m 644 gio/gdbusinterface.h $DESTDIR/include/glib-2.0/gio
install -m 644 gio/gdbusinterfaceskeleton.h $DESTDIR/include/glib-2.0/gio
install -m 644 gio/gdbusintrospection.h $DESTDIR/include/glib-2.0/gio
install -m 644 gio/gdbusmenumodel.h $DESTDIR/include/glib-2.0/gio
install -m 644 gio/gdbusmessage.h $DESTDIR/include/glib-2.0/gio
install -m 644 gio/gdbusmethodinvocation.h $DESTDIR/include/glib-2.0/gio
install -m 644 gio/gdbusnameowning.h $DESTDIR/include/glib-2.0/gio
install -m 644 gio/gdbusnamewatching.h $DESTDIR/include/glib-2.0/gio
install -m 644 gio/gdbusobject.h $DESTDIR/include/glib-2.0/gio
install -m 644 gio/gdbusobjectmanager.h $DESTDIR/include/glib-2.0/gio
install -m 644 gio/gdbusobjectmanagerclient.h $DESTDIR/include/glib-2.0/gio
install -m 644 gio/gdbusobjectmanagerserver.h $DESTDIR/include/glib-2.0/gio
install -m 644 gio/gdbusobjectproxy.h $DESTDIR/include/glib-2.0/gio
install -m 644 gio/gdbusobjectskeleton.h $DESTDIR/include/glib-2.0/gio
install -m 644 gio/gdbusproxy.h $DESTDIR/include/glib-2.0/gio
install -m 644 gio/gdbusserver.h $DESTDIR/include/glib-2.0/gio
install -m 644 gio/gdbusutils.h $DESTDIR/include/glib-2.0/gio
install -m 644 gio/gdrive.h $DESTDIR/include/glib-2.0/gio
install -m 644 gio/gdtlsclientconnection.h $DESTDIR/include/glib-2.0/gio
install -m 644 gio/gdtlsconnection.h $DESTDIR/include/glib-2.0/gio
install -m 644 gio/gdtlsserverconnection.h $DESTDIR/include/glib-2.0/gio
install -m 644 gio/gemblem.h $DESTDIR/include/glib-2.0/gio
install -m 644 gio/gemblemedicon.h $DESTDIR/include/glib-2.0/gio
install -m 644 gio/gfile.h $DESTDIR/include/glib-2.0/gio
install -m 644 gio/gfileattribute.h $DESTDIR/include/glib-2.0/gio
install -m 644 gio/gfileenumerator.h $DESTDIR/include/glib-2.0/gio
install -m 644 gio/gfileicon.h $DESTDIR/include/glib-2.0/gio
install -m 644 gio/gfileinfo.h $DESTDIR/include/glib-2.0/gio
install -m 644 gio/gfileinputstream.h $DESTDIR/include/glib-2.0/gio
install -m 644 gio/gfileiostream.h $DESTDIR/include/glib-2.0/gio
install -m 644 gio/gfilemonitor.h $DESTDIR/include/glib-2.0/gio
install -m 644 gio/gfilenamecompleter.h $DESTDIR/include/glib-2.0/gio
install -m 644 gio/gfileoutputstream.h $DESTDIR/include/glib-2.0/gio
install -m 644 gio/gfilterinputstream.h $DESTDIR/include/glib-2.0/gio
install -m 644 gio/gfilteroutputstream.h $DESTDIR/include/glib-2.0/gio
install -m 644 gio/gicon.h $DESTDIR/include/glib-2.0/gio
install -m 644 gio/ginetaddress.h $DESTDIR/include/glib-2.0/gio
install -m 644 gio/ginetaddressmask.h $DESTDIR/include/glib-2.0/gio
install -m 644 gio/ginetsocketaddress.h $DESTDIR/include/glib-2.0/gio
install -m 644 gio/ginitable.h $DESTDIR/include/glib-2.0/gio
install -m 644 gio/ginputstream.h $DESTDIR/include/glib-2.0/gio
install -m 644 gio/gio-autocleanups.h $DESTDIR/include/glib-2.0/gio
install -m 644 gio/gio.h $DESTDIR/include/glib-2.0/gio
install -m 644 gio/gioenums.h $DESTDIR/include/glib-2.0/gio
install -m 644 gio/gioenumtypes.h $DESTDIR/include/glib-2.0/gio
install -m 644 gio/gioerror.h $DESTDIR/include/glib-2.0/gio
install -m 644 gio/giomodule.h $DESTDIR/include/glib-2.0/gio
install -m 644 gio/gioscheduler.h $DESTDIR/include/glib-2.0/gio
install -m 644 gio/giostream.h $DESTDIR/include/glib-2.0/gio
install -m 644 gio/giotypes.h $DESTDIR/include/glib-2.0/gio
install -m 644 gio/glistmodel.h $DESTDIR/include/glib-2.0/gio
install -m 644 gio/gliststore.h $DESTDIR/include/glib-2.0/gio
install -m 644 gio/gloadableicon.h $DESTDIR/include/glib-2.0/gio
install -m 644 gio/gmemoryinputstream.h $DESTDIR/include/glib-2.0/gio
install -m 644 gio/gmemoryoutputstream.h $DESTDIR/include/glib-2.0/gio
install -m 644 gio/gmenu.h $DESTDIR/include/glib-2.0/gio
install -m 644 gio/gmenuexporter.h $DESTDIR/include/glib-2.0/gio
install -m 644 gio/gmenumodel.h $DESTDIR/include/glib-2.0/gio
install -m 644 gio/gmount.h $DESTDIR/include/glib-2.0/gio
install -m 644 gio/gmountoperation.h $DESTDIR/include/glib-2.0/gio
install -m 644 gio/gnativevolumemonitor.h $DESTDIR/include/glib-2.0/gio
install -m 644 gio/gnetworkaddress.h $DESTDIR/include/glib-2.0/gio
install -m 644 gio/gnetworking.h $DESTDIR/include/glib-2.0/gio
install -m 644 gio/gnetworkmonitor.h $DESTDIR/include/glib-2.0/gio
install -m 644 gio/gnetworkservice.h $DESTDIR/include/glib-2.0/gio
install -m 644 gio/gnotification.h $DESTDIR/include/glib-2.0/gio
install -m 644 gio/goutputstream.h $DESTDIR/include/glib-2.0/gio
install -m 644 gio/gpermission.h $DESTDIR/include/glib-2.0/gio
install -m 644 gio/gpollableinputstream.h $DESTDIR/include/glib-2.0/gio
install -m 644 gio/gpollableoutputstream.h $DESTDIR/include/glib-2.0/gio
install -m 644 gio/gpollableutils.h $DESTDIR/include/glib-2.0/gio
install -m 644 gio/gpropertyaction.h $DESTDIR/include/glib-2.0/gio
install -m 644 gio/gproxy.h $DESTDIR/include/glib-2.0/gio
install -m 644 gio/gproxyaddress.h $DESTDIR/include/glib-2.0/gio
install -m 644 gio/gproxyaddressenumerator.h $DESTDIR/include/glib-2.0/gio
install -m 644 gio/gproxyresolver.h $DESTDIR/include/glib-2.0/gio
install -m 644 gio/gremoteactiongroup.h $DESTDIR/include/glib-2.0/gio
install -m 644 gio/gresolver.h $DESTDIR/include/glib-2.0/gio
install -m 644 gio/gresource.h $DESTDIR/include/glib-2.0/gio
install -m 644 gio/gseekable.h $DESTDIR/include/glib-2.0/gio
install -m 644 gio/gsettings.h $DESTDIR/include/glib-2.0/gio
install -m 644 gio/gsettingsschema.h $DESTDIR/include/glib-2.0/gio
install -m 644 gio/gsimpleaction.h $DESTDIR/include/glib-2.0/gio
install -m 644 gio/gsimpleactiongroup.h $DESTDIR/include/glib-2.0/gio
install -m 644 gio/gsimpleasyncresult.h $DESTDIR/include/glib-2.0/gio
install -m 644 gio/gsimpleiostream.h $DESTDIR/include/glib-2.0/gio
install -m 644 gio/gsimplepermission.h $DESTDIR/include/glib-2.0/gio
install -m 644 gio/gsimpleproxyresolver.h $DESTDIR/include/glib-2.0/gio
install -m 644 gio/gsocket.h $DESTDIR/include/glib-2.0/gio
install -m 644 gio/gsocketaddress.h $DESTDIR/include/glib-2.0/gio
install -m 644 gio/gsocketaddressenumerator.h $DESTDIR/include/glib-2.0/gio
install -m 644 gio/gsocketclient.h $DESTDIR/include/glib-2.0/gio
install -m 644 gio/gsocketconnectable.h $DESTDIR/include/glib-2.0/gio
install -m 644 gio/gsocketconnection.h $DESTDIR/include/glib-2.0/gio
install -m 644 gio/gsocketcontrolmessage.h $DESTDIR/include/glib-2.0/gio
install -m 644 gio/gsocketlistener.h $DESTDIR/include/glib-2.0/gio
install -m 644 gio/gsocketservice.h $DESTDIR/include/glib-2.0/gio
install -m 644 gio/gsrvtarget.h $DESTDIR/include/glib-2.0/gio
install -m 644 gio/gsubprocess.h $DESTDIR/include/glib-2.0/gio
install -m 644 gio/gsubprocesslauncher.h $DESTDIR/include/glib-2.0/gio
install -m 644 gio/gtask.h $DESTDIR/include/glib-2.0/gio
install -m 644 gio/gtcpconnection.h $DESTDIR/include/glib-2.0/gio
install -m 644 gio/gtcpwrapperconnection.h $DESTDIR/include/glib-2.0/gio
install -m 644 gio/gtestdbus.h $DESTDIR/include/glib-2.0/gio
install -m 644 gio/gthemedicon.h $DESTDIR/include/glib-2.0/gio
install -m 644 gio/gthreadedsocketservice.h $DESTDIR/include/glib-2.0/gio
install -m 644 gio/gtlsbackend.h $DESTDIR/include/glib-2.0/gio
install -m 644 gio/gtlscertificate.h $DESTDIR/include/glib-2.0/gio
install -m 644 gio/gtlsclientconnection.h $DESTDIR/include/glib-2.0/gio
install -m 644 gio/gtlsconnection.h $DESTDIR/include/glib-2.0/gio
install -m 644 gio/gtlsdatabase.h $DESTDIR/include/glib-2.0/gio
install -m 644 gio/gtlsfiledatabase.h $DESTDIR/include/glib-2.0/gio
install -m 644 gio/gtlsinteraction.h $DESTDIR/include/glib-2.0/gio
install -m 644 gio/gtlspassword.h $DESTDIR/include/glib-2.0/gio
install -m 644 gio/gtlsserverconnection.h $DESTDIR/include/glib-2.0/gio
install -m 644 gio/gvfs.h $DESTDIR/include/glib-2.0/gio
install -m 644 gio/gvolume.h $DESTDIR/include/glib-2.0/gio
install -m 644 gio/gvolumemonitor.h $DESTDIR/include/glib-2.0/gio
install -m 644 gio/gzlibcompressor.h $DESTDIR/include/glib-2.0/gio
install -m 644 gio/gzlibdecompressor.h $DESTDIR/include/glib-2.0/gio
install -d $DESTDIR/include/glib-2.0/glib
install -m 644 glib/galloca.h $DESTDIR/include/glib-2.0/glib
install -m 644 glib/garray.h $DESTDIR/include/glib-2.0/glib
install -m 644 glib/gasyncqueue.h $DESTDIR/include/glib-2.0/glib
install -m 644 glib/gatomic.h $DESTDIR/include/glib-2.0/glib
install -m 644 glib/gbacktrace.h $DESTDIR/include/glib-2.0/glib
install -m 644 glib/gbase64.h $DESTDIR/include/glib-2.0/glib
install -m 644 glib/gbitlock.h $DESTDIR/include/glib-2.0/glib
install -m 644 glib/gbookmarkfile.h $DESTDIR/include/glib-2.0/glib
install -m 644 glib/gbytes.h $DESTDIR/include/glib-2.0/glib
install -m 644 glib/gcharset.h $DESTDIR/include/glib-2.0/glib
install -m 644 glib/gchecksum.h $DESTDIR/include/glib-2.0/glib
install -m 644 glib/gconvert.h $DESTDIR/include/glib-2.0/glib
install -m 644 glib/gdataset.h $DESTDIR/include/glib-2.0/glib
install -m 644 glib/gdate.h $DESTDIR/include/glib-2.0/glib
install -m 644 glib/gdatetime.h $DESTDIR/include/glib-2.0/glib
install -m 644 glib/gdir.h $DESTDIR/include/glib-2.0/glib
install -m 644 glib/genviron.h $DESTDIR/include/glib-2.0/glib
install -m 644 glib/gerror.h $DESTDIR/include/glib-2.0/glib
install -m 644 glib/gfileutils.h $DESTDIR/include/glib-2.0/glib
install -m 644 glib/ggettext.h $DESTDIR/include/glib-2.0/glib
install -m 644 glib/ghash.h $DESTDIR/include/glib-2.0/glib
install -m 644 glib/ghmac.h $DESTDIR/include/glib-2.0/glib
install -m 644 glib/ghook.h $DESTDIR/include/glib-2.0/glib
install -m 644 glib/ghostutils.h $DESTDIR/include/glib-2.0/glib
install -m 644 glib/gi18n-lib.h $DESTDIR/include/glib-2.0/glib
install -m 644 glib/gi18n.h $DESTDIR/include/glib-2.0/glib
install -m 644 glib/giochannel.h $DESTDIR/include/glib-2.0/glib
install -m 644 glib/gkeyfile.h $DESTDIR/include/glib-2.0/glib
install -m 644 glib/glib-autocleanups.h $DESTDIR/include/glib-2.0/glib
install -m 644 glib/glist.h $DESTDIR/include/glib-2.0/glib
install -m 644 glib/gmacros.h $DESTDIR/include/glib-2.0/glib
install -m 644 glib/gmain.h $DESTDIR/include/glib-2.0/glib
install -m 644 glib/gmappedfile.h $DESTDIR/include/glib-2.0/glib
install -m 644 glib/gmarkup.h $DESTDIR/include/glib-2.0/glib
install -m 644 glib/gmem.h $DESTDIR/include/glib-2.0/glib
install -m 644 glib/gmessages.h $DESTDIR/include/glib-2.0/glib
install -m 644 glib/gnode.h $DESTDIR/include/glib-2.0/glib
install -m 644 glib/goption.h $DESTDIR/include/glib-2.0/glib
install -m 644 glib/gpattern.h $DESTDIR/include/glib-2.0/glib
install -m 644 glib/gpoll.h $DESTDIR/include/glib-2.0/glib
install -m 644 glib/gprimes.h $DESTDIR/include/glib-2.0/glib
install -m 644 glib/gprintf.h $DESTDIR/include/glib-2.0/glib
install -m 644 glib/gqsort.h $DESTDIR/include/glib-2.0/glib
install -m 644 glib/gquark.h $DESTDIR/include/glib-2.0/glib
install -m 644 glib/gqueue.h $DESTDIR/include/glib-2.0/glib
install -m 644 glib/grand.h $DESTDIR/include/glib-2.0/glib
install -m 644 glib/gregex.h $DESTDIR/include/glib-2.0/glib
install -m 644 glib/gscanner.h $DESTDIR/include/glib-2.0/glib
install -m 644 glib/gsequence.h $DESTDIR/include/glib-2.0/glib
install -m 644 glib/gshell.h $DESTDIR/include/glib-2.0/glib
install -m 644 glib/gslice.h $DESTDIR/include/glib-2.0/glib
install -m 644 glib/gslist.h $DESTDIR/include/glib-2.0/glib
install -m 644 glib/gspawn.h $DESTDIR/include/glib-2.0/glib
install -m 644 glib/gstdio.h $DESTDIR/include/glib-2.0/glib
install -m 644 glib/gstrfuncs.h $DESTDIR/include/glib-2.0/glib
install -m 644 glib/gstring.h $DESTDIR/include/glib-2.0/glib
install -m 644 glib/gstringchunk.h $DESTDIR/include/glib-2.0/glib
install -m 644 glib/gtestutils.h $DESTDIR/include/glib-2.0/glib
install -m 644 glib/gthread.h $DESTDIR/include/glib-2.0/glib
install -m 644 glib/gthreadpool.h $DESTDIR/include/glib-2.0/glib
install -m 644 glib/gtimer.h $DESTDIR/include/glib-2.0/glib
install -m 644 glib/gtimezone.h $DESTDIR/include/glib-2.0/glib
install -m 644 glib/gtrashstack.h $DESTDIR/include/glib-2.0/glib
install -m 644 glib/gtree.h $DESTDIR/include/glib-2.0/glib
install -m 644 glib/gtypes.h $DESTDIR/include/glib-2.0/glib
install -m 644 glib/gunicode.h $DESTDIR/include/glib-2.0/glib
install -m 644 glib/gurifuncs.h $DESTDIR/include/glib-2.0/glib
install -m 644 glib/gutils.h $DESTDIR/include/glib-2.0/glib
install -m 644 glib/guuid.h $DESTDIR/include/glib-2.0/glib
install -m 644 glib/gvariant.h $DESTDIR/include/glib-2.0/glib
install -m 644 glib/gvarianttype.h $DESTDIR/include/glib-2.0/glib
install -m 644 glib/gversion.h $DESTDIR/include/glib-2.0/glib
install -m 644 glib/gversionmacros.h $DESTDIR/include/glib-2.0/glib
install -d $DESTDIR/include/glib-2.0/glib/deprecated
install -m 644 glib/deprecated/gallocator.h $DESTDIR/include/glib-2.0/glib/deprecated
install -m 644 glib/deprecated/gcache.h $DESTDIR/include/glib-2.0/glib/deprecated
install -m 644 glib/deprecated/gcompletion.h $DESTDIR/include/glib-2.0/glib/deprecated
install -m 644 glib/deprecated/gmain.h $DESTDIR/include/glib-2.0/glib/deprecated
install -m 644 glib/deprecated/grel.h $DESTDIR/include/glib-2.0/glib/deprecated
install -m 644 glib/deprecated/gthread.h $DESTDIR/include/glib-2.0/glib/deprecated
install -d $DESTDIR/include/glib-2.0/gobject
install -m 644 gobject/gbinding.h $DESTDIR/include/glib-2.0/gobject
install -m 644 gobject/gboxed.h $DESTDIR/include/glib-2.0/gobject
install -m 644 gobject/gclosure.h $DESTDIR/include/glib-2.0/gobject
install -m 644 gobject/genums.h $DESTDIR/include/glib-2.0/gobject
install -m 644 gobject/glib-types.h $DESTDIR/include/glib-2.0/gobject
install -m 644 gobject/gmarshal.h $DESTDIR/include/glib-2.0/gobject
install -m 644 gobject/gobject-autocleanups.h $DESTDIR/include/glib-2.0/gobject
install -m 644 gobject/gobject.h $DESTDIR/include/glib-2.0/gobject
install -m 644 gobject/gobjectnotifyqueue.c $DESTDIR/include/glib-2.0/gobject
install -m 644 gobject/gparam.h $DESTDIR/include/glib-2.0/gobject
install -m 644 gobject/gparamspecs.h $DESTDIR/include/glib-2.0/gobject
install -m 644 gobject/gsignal.h $DESTDIR/include/glib-2.0/gobject
install -m 644 gobject/gsourceclosure.h $DESTDIR/include/glib-2.0/gobject
install -m 644 gobject/gtype.h $DESTDIR/include/glib-2.0/gobject
install -m 644 gobject/gtypemodule.h $DESTDIR/include/glib-2.0/gobject
install -m 644 gobject/gtypeplugin.h $DESTDIR/include/glib-2.0/gobject
install -m 644 gobject/gvalue.h $DESTDIR/include/glib-2.0/gobject
install -m 644 gobject/gvaluearray.h $DESTDIR/include/glib-2.0/gobject
install -m 644 gobject/gvaluecollector.h $DESTDIR/include/glib-2.0/gobject
install -m 644 gobject/gvaluetypes.h $DESTDIR/include/glib-2.0/gobject
install -d $DESTDIR/lib
install gio/.libs/libgio-2.0.so.0 $DESTDIR/lib
ln -sf libgio-2.0.so.0 $DESTDIR/lib/libgio-2.0.so
install glib/.libs/libglib-2.0.so.0 $DESTDIR/lib
ln -sf libglib-2.0.so.0 $DESTDIR/lib/libglib-2.0.so
install gmodule/.libs/libgmodule-2.0.so.0 $DESTDIR/lib
ln -sf libgmodule-2.0.so.0 $DESTDIR/lib/libgmodule-2.0.so
install gobject/.libs/libgobject-2.0.so.0 $DESTDIR/lib
ln -sf libgobject-2.0.so.0 $DESTDIR/lib/libgobject-2.0.so
install gthread/.libs/libgthread-2.0.so.0 $DESTDIR/lib
ln -sf libgthread-2.0.so.0 $DESTDIR/lib/libgthread-2.0.so
install -d $DESTDIR/lib/pkgconfig
install -m 644 gio-2.0.pc $DESTDIR/lib/pkgconfig
install -m 644 gio-unix-2.0.pc $DESTDIR/lib/pkgconfig
install -m 644 glib-2.0.pc $DESTDIR/lib/pkgconfig
install -m 644 gmodule-2.0.pc $DESTDIR/lib/pkgconfig
install -m 644 gmodule-no-export-2.0.pc $DESTDIR/lib/pkgconfig
install -m 644 gobject-2.0.pc $DESTDIR/lib/pkgconfig
install -m 644 gthread-2.0.pc $DESTDIR/lib/pkgconfig
install -d $DESTDIR/share/aclocal
install -m 644 m4macros/glib-2.0.m4 $DESTDIR/share/aclocal
install -m 644 m4macros/glib-gettext.m4 $DESTDIR/share/aclocal
install -m 644 m4macros/gsettings.m4 $DESTDIR/share/aclocal
install -d $DESTDIR/share/glib-2.0/codegen
install -m 644 gio/gdbus-2.0/codegen/__init__.py $DESTDIR/share/glib-2.0/codegen
install -m 644 gio/gdbus-2.0/codegen/codegen.py $DESTDIR/share/glib-2.0/codegen
install -m 644 gio/gdbus-2.0/codegen/codegen_docbook.py $DESTDIR/share/glib-2.0/codegen
install -m 644 gio/gdbus-2.0/codegen/codegen_main.py $DESTDIR/share/glib-2.0/codegen
install -m 644 gio/gdbus-2.0/codegen/config.py $DESTDIR/share/glib-2.0/codegen
install -m 644 gio/gdbus-2.0/codegen/dbustypes.py $DESTDIR/share/glib-2.0/codegen
install -m 644 gio/gdbus-2.0/codegen/parser.py $DESTDIR/share/glib-2.0/codegen
install -m 644 gio/gdbus-2.0/codegen/utils.py $DESTDIR/share/glib-2.0/codegen

# build.pc
install -d $DESTDIR/share/pkgconfig
cat > $DESTDIR/share/pkgconfig/glib-build.pc << EOF
Name: glib
Version: 2.52.3
Description: GLib library
Requires: $REQUIRES

devel=\\
/bin/gdbus-codegen \\
/bin/glib-compile-resources \\
/bin/glib-compile-schemas \\
/bin/glib-genmarshal \\
/bin/glib-mkenums \\
/include/gio-unix-2.0/gio/gdesktopappinfo.h \\
/include/gio-unix-2.0/gio/gunixconnection.h \\
/include/gio-unix-2.0/gio/gunixfdlist.h \\
/include/gio-unix-2.0/gio/gunixfdmessage.h \\
/include/gio-unix-2.0/gio/gunixinputstream.h \\
/include/gio-unix-2.0/gio/gunixmounts.h \\
/include/gio-unix-2.0/gio/gunixoutputstream.h \\
/include/gio-unix-2.0/gio/gunixsocketaddress.h \\
/include/glib-2.0/glib.h \\
/include/glib-2.0/glibconfig.h \\
/include/glib-2.0/glib-object.h \\
/include/glib-2.0/glib-unix.h \\
/include/glib-2.0/gmodule.h \\
/include/glib-2.0/gio/gaction.h \\
/include/glib-2.0/gio/gactiongroup.h \\
/include/glib-2.0/gio/gactiongroupexporter.h \\
/include/glib-2.0/gio/gactionmap.h \\
/include/glib-2.0/gio/gappinfo.h \\
/include/glib-2.0/gio/gapplication.h \\
/include/glib-2.0/gio/gapplicationcommandline.h \\
/include/glib-2.0/gio/gasyncinitable.h \\
/include/glib-2.0/gio/gasyncresult.h \\
/include/glib-2.0/gio/gbufferedinputstream.h \\
/include/glib-2.0/gio/gbufferedoutputstream.h \\
/include/glib-2.0/gio/gbytesicon.h \\
/include/glib-2.0/gio/gcancellable.h \\
/include/glib-2.0/gio/gcharsetconverter.h \\
/include/glib-2.0/gio/gcontenttype.h \\
/include/glib-2.0/gio/gconverter.h \\
/include/glib-2.0/gio/gconverterinputstream.h \\
/include/glib-2.0/gio/gconverteroutputstream.h \\
/include/glib-2.0/gio/gcredentials.h \\
/include/glib-2.0/gio/gdatagrambased.h \\
/include/glib-2.0/gio/gdatainputstream.h \\
/include/glib-2.0/gio/gdataoutputstream.h \\
/include/glib-2.0/gio/gdbusactiongroup.h \\
/include/glib-2.0/gio/gdbusaddress.h \\
/include/glib-2.0/gio/gdbusauthobserver.h \\
/include/glib-2.0/gio/gdbusconnection.h \\
/include/glib-2.0/gio/gdbuserror.h \\
/include/glib-2.0/gio/gdbusinterface.h \\
/include/glib-2.0/gio/gdbusinterfaceskeleton.h \\
/include/glib-2.0/gio/gdbusintrospection.h \\
/include/glib-2.0/gio/gdbusmenumodel.h \\
/include/glib-2.0/gio/gdbusmessage.h \\
/include/glib-2.0/gio/gdbusmethodinvocation.h \\
/include/glib-2.0/gio/gdbusnameowning.h \\
/include/glib-2.0/gio/gdbusnamewatching.h \\
/include/glib-2.0/gio/gdbusobject.h \\
/include/glib-2.0/gio/gdbusobjectmanager.h \\
/include/glib-2.0/gio/gdbusobjectmanagerclient.h \\
/include/glib-2.0/gio/gdbusobjectmanagerserver.h \\
/include/glib-2.0/gio/gdbusobjectproxy.h \\
/include/glib-2.0/gio/gdbusobjectskeleton.h \\
/include/glib-2.0/gio/gdbusproxy.h \\
/include/glib-2.0/gio/gdbusserver.h \\
/include/glib-2.0/gio/gdbusutils.h \\
/include/glib-2.0/gio/gdrive.h \\
/include/glib-2.0/gio/gdtlsclientconnection.h \\
/include/glib-2.0/gio/gdtlsconnection.h \\
/include/glib-2.0/gio/gdtlsserverconnection.h \\
/include/glib-2.0/gio/gemblem.h \\
/include/glib-2.0/gio/gemblemedicon.h \\
/include/glib-2.0/gio/gfile.h \\
/include/glib-2.0/gio/gfileattribute.h \\
/include/glib-2.0/gio/gfileenumerator.h \\
/include/glib-2.0/gio/gfileicon.h \\
/include/glib-2.0/gio/gfileinfo.h \\
/include/glib-2.0/gio/gfileinputstream.h \\
/include/glib-2.0/gio/gfileiostream.h \\
/include/glib-2.0/gio/gfilemonitor.h \\
/include/glib-2.0/gio/gfilenamecompleter.h \\
/include/glib-2.0/gio/gfileoutputstream.h \\
/include/glib-2.0/gio/gfilterinputstream.h \\
/include/glib-2.0/gio/gfilteroutputstream.h \\
/include/glib-2.0/gio/gicon.h \\
/include/glib-2.0/gio/ginetaddress.h \\
/include/glib-2.0/gio/ginetaddressmask.h \\
/include/glib-2.0/gio/ginetsocketaddress.h \\
/include/glib-2.0/gio/ginitable.h \\
/include/glib-2.0/gio/ginputstream.h \\
/include/glib-2.0/gio/gio-autocleanups.h \\
/include/glib-2.0/gio/gio.h \\
/include/glib-2.0/gio/gioenums.h \\
/include/glib-2.0/gio/gioenumtypes.h \\
/include/glib-2.0/gio/gioerror.h \\
/include/glib-2.0/gio/giomodule.h \\
/include/glib-2.0/gio/gioscheduler.h \\
/include/glib-2.0/gio/giostream.h \\
/include/glib-2.0/gio/giotypes.h \\
/include/glib-2.0/gio/glistmodel.h \\
/include/glib-2.0/gio/gliststore.h \\
/include/glib-2.0/gio/gloadableicon.h \\
/include/glib-2.0/gio/gmemoryinputstream.h \\
/include/glib-2.0/gio/gmemoryoutputstream.h \\
/include/glib-2.0/gio/gmenu.h \\
/include/glib-2.0/gio/gmenuexporter.h \\
/include/glib-2.0/gio/gmenumodel.h \\
/include/glib-2.0/gio/gmount.h \\
/include/glib-2.0/gio/gmountoperation.h \\
/include/glib-2.0/gio/gnativevolumemonitor.h \\
/include/glib-2.0/gio/gnetworkaddress.h \\
/include/glib-2.0/gio/gnetworking.h \\
/include/glib-2.0/gio/gnetworkmonitor.h \\
/include/glib-2.0/gio/gnetworkservice.h \\
/include/glib-2.0/gio/gnotification.h \\
/include/glib-2.0/gio/goutputstream.h \\
/include/glib-2.0/gio/gpermission.h \\
/include/glib-2.0/gio/gpollableinputstream.h \\
/include/glib-2.0/gio/gpollableoutputstream.h \\
/include/glib-2.0/gio/gpollableutils.h \\
/include/glib-2.0/gio/gpropertyaction.h \\
/include/glib-2.0/gio/gproxy.h \\
/include/glib-2.0/gio/gproxyaddress.h \\
/include/glib-2.0/gio/gproxyaddressenumerator.h \\
/include/glib-2.0/gio/gproxyresolver.h \\
/include/glib-2.0/gio/gremoteactiongroup.h \\
/include/glib-2.0/gio/gresolver.h \\
/include/glib-2.0/gio/gresource.h \\
/include/glib-2.0/gio/gseekable.h \\
/include/glib-2.0/gio/gsettings.h \\
/include/glib-2.0/gio/gsettingsschema.h \\
/include/glib-2.0/gio/gsimpleaction.h \\
/include/glib-2.0/gio/gsimpleactiongroup.h \\
/include/glib-2.0/gio/gsimpleasyncresult.h \\
/include/glib-2.0/gio/gsimpleiostream.h \\
/include/glib-2.0/gio/gsimplepermission.h \\
/include/glib-2.0/gio/gsimpleproxyresolver.h \\
/include/glib-2.0/gio/gsocket.h \\
/include/glib-2.0/gio/gsocketaddress.h \\
/include/glib-2.0/gio/gsocketaddressenumerator.h \\
/include/glib-2.0/gio/gsocketclient.h \\
/include/glib-2.0/gio/gsocketconnectable.h \\
/include/glib-2.0/gio/gsocketconnection.h \\
/include/glib-2.0/gio/gsocketcontrolmessage.h \\
/include/glib-2.0/gio/gsocketlistener.h \\
/include/glib-2.0/gio/gsocketservice.h \\
/include/glib-2.0/gio/gsrvtarget.h \\
/include/glib-2.0/gio/gsubprocess.h \\
/include/glib-2.0/gio/gsubprocesslauncher.h \\
/include/glib-2.0/gio/gtask.h \\
/include/glib-2.0/gio/gtcpconnection.h \\
/include/glib-2.0/gio/gtcpwrapperconnection.h \\
/include/glib-2.0/gio/gtestdbus.h \\
/include/glib-2.0/gio/gthemedicon.h \\
/include/glib-2.0/gio/gthreadedsocketservice.h \\
/include/glib-2.0/gio/gtlsbackend.h \\
/include/glib-2.0/gio/gtlscertificate.h \\
/include/glib-2.0/gio/gtlsclientconnection.h \\
/include/glib-2.0/gio/gtlsconnection.h \\
/include/glib-2.0/gio/gtlsdatabase.h \\
/include/glib-2.0/gio/gtlsfiledatabase.h \\
/include/glib-2.0/gio/gtlsinteraction.h \\
/include/glib-2.0/gio/gtlspassword.h \\
/include/glib-2.0/gio/gtlsserverconnection.h \\
/include/glib-2.0/gio/gvfs.h \\
/include/glib-2.0/gio/gvolume.h \\
/include/glib-2.0/gio/gvolumemonitor.h \\
/include/glib-2.0/gio/gzlibcompressor.h \\
/include/glib-2.0/gio/gzlibdecompressor.h \\
/include/glib-2.0/glib/galloca.h \\
/include/glib-2.0/glib/garray.h \\
/include/glib-2.0/glib/gasyncqueue.h \\
/include/glib-2.0/glib/gatomic.h \\
/include/glib-2.0/glib/gbacktrace.h \\
/include/glib-2.0/glib/gbase64.h \\
/include/glib-2.0/glib/gbitlock.h \\
/include/glib-2.0/glib/gbookmarkfile.h \\
/include/glib-2.0/glib/gbytes.h \\
/include/glib-2.0/glib/gcharset.h \\
/include/glib-2.0/glib/gchecksum.h \\
/include/glib-2.0/glib/gconvert.h \\
/include/glib-2.0/glib/gdataset.h \\
/include/glib-2.0/glib/gdate.h \\
/include/glib-2.0/glib/gdatetime.h \\
/include/glib-2.0/glib/gdir.h \\
/include/glib-2.0/glib/genviron.h \\
/include/glib-2.0/glib/gerror.h \\
/include/glib-2.0/glib/gfileutils.h \\
/include/glib-2.0/glib/ggettext.h \\
/include/glib-2.0/glib/ghash.h \\
/include/glib-2.0/glib/ghmac.h \\
/include/glib-2.0/glib/ghook.h \\
/include/glib-2.0/glib/ghostutils.h \\
/include/glib-2.0/glib/gi18n-lib.h \\
/include/glib-2.0/glib/gi18n.h \\
/include/glib-2.0/glib/giochannel.h \\
/include/glib-2.0/glib/gkeyfile.h \\
/include/glib-2.0/glib/glib-autocleanups.h \\
/include/glib-2.0/glib/glist.h \\
/include/glib-2.0/glib/gmacros.h \\
/include/glib-2.0/glib/gmain.h \\
/include/glib-2.0/glib/gmappedfile.h \\
/include/glib-2.0/glib/gmarkup.h \\
/include/glib-2.0/glib/gmem.h \\
/include/glib-2.0/glib/gmessages.h \\
/include/glib-2.0/glib/gnode.h \\
/include/glib-2.0/glib/goption.h \\
/include/glib-2.0/glib/gpattern.h \\
/include/glib-2.0/glib/gpoll.h \\
/include/glib-2.0/glib/gprimes.h \\
/include/glib-2.0/glib/gprintf.h \\
/include/glib-2.0/glib/gqsort.h \\
/include/glib-2.0/glib/gquark.h \\
/include/glib-2.0/glib/gqueue.h \\
/include/glib-2.0/glib/grand.h \\
/include/glib-2.0/glib/gregex.h \\
/include/glib-2.0/glib/gscanner.h \\
/include/glib-2.0/glib/gsequence.h \\
/include/glib-2.0/glib/gshell.h \\
/include/glib-2.0/glib/gslice.h \\
/include/glib-2.0/glib/gslist.h \\
/include/glib-2.0/glib/gspawn.h \\
/include/glib-2.0/glib/gstdio.h \\
/include/glib-2.0/glib/gstrfuncs.h \\
/include/glib-2.0/glib/gstring.h \\
/include/glib-2.0/glib/gstringchunk.h \\
/include/glib-2.0/glib/gtestutils.h \\
/include/glib-2.0/glib/gthread.h \\
/include/glib-2.0/glib/gthreadpool.h \\
/include/glib-2.0/glib/gtimer.h \\
/include/glib-2.0/glib/gtimezone.h \\
/include/glib-2.0/glib/gtrashstack.h \\
/include/glib-2.0/glib/gtree.h \\
/include/glib-2.0/glib/gtypes.h \\
/include/glib-2.0/glib/gunicode.h \\
/include/glib-2.0/glib/gurifuncs.h \\
/include/glib-2.0/glib/gutils.h \\
/include/glib-2.0/glib/guuid.h \\
/include/glib-2.0/glib/gvariant.h \\
/include/glib-2.0/glib/gvarianttype.h \\
/include/glib-2.0/glib/gversion.h \\
/include/glib-2.0/glib/gversionmacros.h \\
/include/glib-2.0/glib/deprecated/gallocator.h \\
/include/glib-2.0/glib/deprecated/gcache.h \\
/include/glib-2.0/glib/deprecated/gcompletion.h \\
/include/glib-2.0/glib/deprecated/gmain.h \\
/include/glib-2.0/glib/deprecated/grel.h \\
/include/glib-2.0/glib/deprecated/gthread.h \\
/include/glib-2.0/gobject/gbinding.h \\
/include/glib-2.0/gobject/gboxed.h \\
/include/glib-2.0/gobject/gclosure.h \\
/include/glib-2.0/gobject/genums.h \\
/include/glib-2.0/gobject/glib-types.h \\
/include/glib-2.0/gobject/gmarshal.h \\
/include/glib-2.0/gobject/gobject-autocleanups.h \\
/include/glib-2.0/gobject/gobject.h \\
/include/glib-2.0/gobject/gobjectnotifyqueue.c \\
/include/glib-2.0/gobject/gparam.h \\
/include/glib-2.0/gobject/gparamspecs.h \\
/include/glib-2.0/gobject/gsignal.h \\
/include/glib-2.0/gobject/gsourceclosure.h \\
/include/glib-2.0/gobject/gtype.h \\
/include/glib-2.0/gobject/gtypemodule.h \\
/include/glib-2.0/gobject/gtypeplugin.h \\
/include/glib-2.0/gobject/gvalue.h \\
/include/glib-2.0/gobject/gvaluearray.h \\
/include/glib-2.0/gobject/gvaluecollector.h \\
/include/glib-2.0/gobject/gvaluetypes.h \\
/lib/libgio-2.0.so \\
/lib/libglib-2.0.so \\
/lib/libgmodule-2.0.so \\
/lib/libgobject-2.0.so \\
/lib/libgthread-2.0.so \\
/lib/pkgconfig/gio-2.0.pc \\
/lib/pkgconfig/gio-unix-2.0.pc \\
/lib/pkgconfig/glib-2.0.pc \\
/lib/pkgconfig/gmodule-2.0.pc \\
/lib/pkgconfig/gmodule-no-export-2.0.pc \\
/lib/pkgconfig/gobject-2.0.pc \\
/lib/pkgconfig/gthread-2.0.pc \\
/share/aclocal/glib-2.0.m4 \\
/share/aclocal/glib-gettext.m4 \\
/share/aclocal/gsettings.m4 \\
/share/glib-2.0/codegen/__init__.py \\
/share/glib-2.0/codegen/codegen.py \\
/share/glib-2.0/codegen/codegen_docbook.py \\
/share/glib-2.0/codegen/codegen_main.py \\
/share/glib-2.0/codegen/config.py \\
/share/glib-2.0/codegen/dbustypes.py \\
/share/glib-2.0/codegen/parser.py \\
/share/glib-2.0/codegen/utils.py

exec=\\
/bin/gapplication \\
/bin/gdbus \\
/bin/gio \\
/bin/gio-querymodules \\
/bin/gresource \\
/bin/gsettings \\
/lib/libgio-2.0.so.0 \\
/lib/libglib-2.0.so.0 \\
/lib/libgmodule-2.0.so.0 \\
/lib/libgobject-2.0.so.0 \\
/lib/libgthread-2.0.so.0
EOF

exit

# patch
--- glib-2.52.3.orig/configure.ac
+++ glib-2.52.3/configure.ac
@@ -63,8 +63,8 @@
 # Save this value here, since automake will set cflags later
 cflags_set=${CFLAGS:+set}
 
-AM_INIT_AUTOMAKE([1.11 -Wno-portability no-define no-dist-gzip dist-xz tar-ustar])
-AM_MAINTAINER_MODE([enable])
+AM_INIT_AUTOMAKE([1.11 -Wno-portability no-define no-dist-gzip dist-xz tar-ustar subdir-objects])
+AM_MAINTAINER_MODE
 
 # Support silent build rules. Disable
 # by either passing --disable-silent-rules to configure or passing V=1
@@ -463,13 +463,6 @@
 AC_SUBST([CONFIG_STATUS_DEPENDENCIES],['$(top_srcdir)/po/LINGUAS'])
 GLIB_GNU_GETTEXT
 
-if test "$gt_cv_have_gettext" != "yes" ; then
-  AC_MSG_ERROR([
-*** You must have either have gettext support in your C library, or use the 
-*** GNU gettext library. (http://www.gnu.org/software/gettext/gettext.html
-])
-fi
-
 LIBS="$INTLLIBS $LIBS"
 
 GETTEXT_PACKAGE=glib20
--- glib-2.52.3.orig/gio/tests/Makefile.am
+++ glib-2.52.3/gio/tests/Makefile.am
@@ -232,10 +232,22 @@
 
 uninstalled_test_extra_programs += gdbus-daemon
 nodist_gdbus_daemon_SOURCES = \
-	$(top_builddir)/gio/gdbus-daemon-generated.c
-gdbus_daemon_SOURCES = \
-	gdbus-daemon.c				\
-	$(top_srcdir)/gio/gdbusdaemon.c
+	gdbus-daemon-generated.c \
+	gdbus-daemon-impl.c
+gdbus_daemon_SOURCES = gdbus-daemon.c
+
+gdbus-daemon-impl.c: $(top_srcdir)/gio/gdbusdaemon.c
+	$(AM_V_GEN) $(LN_S) $^ $@
+
+gdbus-daemon-generated.h gdbus-daemon-generated.c: $(top_srcdir)/gio/dbus-daemon.xml $(GDBUS_PYTHON_DEPS)
+	$(AM_V_GEN) UNINSTALLED_GLIB_SRCDIR=$(top_srcdir) \
+		UNINSTALLED_GLIB_BUILDDIR=$(top_builddir) \
+		$(PYTHON) $(top_srcdir)/gio/gdbus-2.0/codegen/gdbus-codegen.in \
+		--interface-prefix org. \
+		--generate-c-code gdbus-daemon-generated \
+		--c-namespace _G \
+		$(top_srcdir)/gio/dbus-daemon.xml \
+		$(NULL)
 
 # -----------------------------------------------------------------------------
 #  Test programs buildable on UNIX only
--- glib-2.52.3.orig/glib/Makefile.am
+++ glib-2.52.3/glib/Makefile.am
@@ -11,8 +11,7 @@
 #
 DISTCLEANFILES += glibconfig-stamp glibconfig.h glibconfig.h.win32
 BUILT_SOURCES += glibconfig-stamp
-configexecincludedir = $(libdir)/glib-2.0/include
-nodist_configexecinclude_HEADERS = glibconfig.h
+nodist_glibinclude_HEADERS = glibconfig.h
 glibconfig-stamp: ../config.status
 	$(AM_V_GEN) cd $(top_builddir) && \
 	  $(SHELL) ./config.status glib/glibconfig.h
--- glib-2.52.3.orig/glib/ggettext.c
+++ glib-2.52.3/glib/ggettext.c
@@ -40,7 +40,6 @@
 
 #include <string.h>
 #include <locale.h>
-#include <libintl.h>
 
 #ifdef G_OS_WIN32
 
--- glib-2.52.3.orig/glib/glibintl.h
+++ glib-2.52.3/glib/glibintl.h
@@ -33,6 +33,7 @@
 #define gettext(String) (String)
 #define dgettext(Domain,String) (String)
 #define dcgettext(Domain,String,Type) (String)
+#define ngettext(String1,String2,N) dngettext(NULL,String1,String2,N)
 #define dngettext(Domain,String1,String2,N) ((N) == 1 ? (String1) : (String2))
 #define bindtextdomain(Domain,Directory) (Domain) 
 #define bind_textdomain_codeset(Domain,Codeset)
--- glib-2.52.3.orig/glib-2.0.pc.in
+++ glib-2.52.3/glib-2.0.pc.in
@@ -13,4 +13,4 @@
 Requires.private: @PCRE_REQUIRES@
 Libs: -L${libdir} -lglib-2.0 @INTLLIBS@
 Libs.private: @G_THREAD_LIBS@ @G_LIBS_EXTRA@ @PCRE_LIBS@ @INTLLIBS@ @ICONV_LIBS@ @CARBON_LIBS@ @COCOA_LIBS@
-Cflags: -I${includedir}/glib-2.0 -I${libdir}/glib-2.0/include @GLIB_EXTRA_CFLAGS@
+Cflags: -I${includedir}/glib-2.0 @GLIB_EXTRA_CFLAGS@
--- glib-2.52.3.orig/win32/vs15/Makefile.am
+++ glib-2.52.3/win32/vs15/Makefile.am
@@ -44,6 +44,5 @@
 MSVC_BASE_VER_LONG = 2010
 MSVC_VER = 15
 MSVC_VER_LONG = 15
-MSVC_TOOLSET = 141
 
 include $(top_srcdir)/win32/Makefile-newvs.am

# source
https://download.gnome.org/sources/glib/2.52/glib-2.52.3.tar.xz
